geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingFunction.java

```diff
@@ -75,9 +77,19 @@ public CliFunctionResult executeFunction(FunctionContext<Object[]> context)
    * and the given async-event-queue as one of its queues.
    */
   private void alterRegion(Region<?, ?> region, String queueName, boolean synchronous) {
-    region.getAttributesMutator().setCacheLoader(new JdbcLoader());
+    boolean isProxy = false;
+    if (region instanceof PartitionedRegion) {
```

Comment:
In general this seems wrong. I think something has gone wrong at a higher level if we have proxies here.
We should not even have executed this function on this member.
Why would we still create the async queue on a proxy member?
We check for cluster config first thing to see if we can create the mapping on the region back in CreateRegionCommand. Seems like we should have checked their if it was a proxy region. If it is then we fail.
Otherwise when we ask for all the members using that cluster config (either the cluster wide one or the group) shouldn't they all have the region as configured in the cluster config? Then at this point it could not even be a proxy.
If you are using cluster config (which create jdbc-mapping requires) then don't you need to put the proxy definition in a separate group? So how are we even executing this function on servers that have a proxy? I think the fix should be in CreateMappingCommand before we call any function

Reply:
I tried your suggestion and worked out a version to avoiding calling function for accessor member. Then I realized this member need to set AEQ_id, so the function still need to be called. Otherwise, it will fail later with "GatewaySenderConfigurationException: Region DefaultRegion has [JDBC#DefaultRegion] AsyncEvent queue IDs. Another cache has same region with [] AsyncEvent queue IDs. For region across all members, AsyncEvent queue IDs should be same." 
So I will switch back to my previous solution and refine it.